infra/ansible/playbooks/k8s/preemptive-worker.yml

```diff
@@ -50,4 +50,4 @@
       cron:
         name: "Reset k8s node"
         minute: "9"
-        job: "/bin/bash /root/leave_k8s_cluster.sh"
+        job: "sudo /bin/bash /root/leave_k8s_cluster.sh"
```

Comment:
[run leave script as super user](https://github.com/airqo-platform/AirQo-api/pull/1327/commits/1df00867baefb9b5c96d581bc2f4cdb52366dc9b)

Reply:
Thanks @123MwanjeMike , much appreciated!
What challenge is this super user privilege addressing?

Reply:
Hi @Baalmart 
The challenge was that the leave script was running the `kubeadm reset` command which can only be run with administrative rights.